event-log/README.md

```diff
@@ -0,0 +1,243 @@
+# event-log
+
+This is a microservice which provides CRUD operations for Event Log DB.
+
+## API
+
+| Method | Path                                     | Description                                                    |
+|--------|------------------------------------------|----------------------------------------------------------------|
+|  POST  | ```/events```                            | Creates an event with a `NEW` status                           |
+|  PATCH | ```/events```                            | Changes events' data by applying the given patch               |
```

Comment:
I think this should be `PATCH /events/:id`

Reply:
Yes, this endpoint modifies all the events.

Reply:
Ok - in this case I wouldn't be too dogmatic and leave it as you suggested.